examples/jupyter/expansion-filters.ipynb

```diff
@@ -437,6 +439,1116 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "f1 = plt.imread('flux3d.png')\n",
```

Comment:
You should add this file to version control too. Otherwise, if one of us needs to re-run the notebook, this last cell won't work because the file won't be present.

Reply:
Thanks. @paulromano. All your comments are addressed.